### PR TITLE
Data visulaization of nomic data 'figure 2' maybe

### PR DESCRIPTION
**Pull request by MikeLippincott**
Here I visualized selected treatments' cytokines a few different ways

### CODE REVIEW

**Review by MattsonCam — APPROVED** [review]
LGTM @MikeLippincott, maybe just some minor adjustments

File: 0.Preprocess_Data/1.preprocessing.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (MattsonCam) [inline 0.Preprocess_Data/1.preprocessing.ipynb:2]: <p>Good documentation here</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

File: 1.Exploratory_Data_Analysis/1.umap_analysis_plate2.ipynb [hunk 2]
```diff
@@ -43,19 +43,13 @@
      "name": "stderr",
```
Comment (MattsonCam) [inline 1.Exploratory_Data_Analysis/1.umap_analysis_plate2.ipynb:43]: <pre class="ql-syntax" spellcheck="false">Line #9.    # df = df.sample(n=1000, random_state=0, weights='fourb_Metadata_Treatment_Dose_Inhibitor_Dose')
</pre><p>May have not seen it, but consider removing this code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

File: 1.Exploratory_Data_Analysis/2.single_channel_umap_analysis.ipynb [hunk 3]
```diff
@@ -0,0 +1,598 @@
+{
```
Comment (MattsonCam) [inline 1.Exploratory_Data_Analysis/2.single_channel_umap_analysis.ipynb]: <p>May also consider storing these in a dictionary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: 2.Nomic_nELISA_Analysis/0.Exploratory_Analysis/PBMC/3.cytokine_visualization.ipynb [hunk 4]
```diff
@@ -0,0 +1,2239 @@
+{
```
Comment (MattsonCam) [inline 2.Nomic_nELISA_Analysis/0.Exploratory_Analysis/PBMC/3.cytokine_visualization.ipynb]: <p>Didn't know you could use ggplot in python, cool!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

File: 1.Exploratory_Data_Analysis/1.umap_analysis_plate2.ipynb (hunk 2 above)
Comment (MikeLippincott) [reply 1.Exploratory_Data_Analysis/1.umap_analysis_plate2.ipynb:43]: removed line

**Pull request merged**